cg/apps/lims/sample_sheet.py

```diff
@@ -67,14 +72,12 @@ def get_index(lims: Lims, label: str) -> str:
 
 
 def flow_cell_samples(
-    lims: Lims, flowcell_id: str, bcl_converter: str
-) -> Iterable[Union[FlowCellSampleBcl2Fastq, FlowCellSampleDragen]]:
-    lims_flowcell_sample = {
-        BclConverter.BCL2FASTQ.value: FlowCellSampleBcl2Fastq,
-        BclConverter.DRAGEN.value: FlowCellSampleDragen,
-    }
-    LOG.info(f"Fetching samples from lims for flowcell {flowcell_id}")
-    containers: List[Container] = lims.get_containers(name=flowcell_id)
+    lims: Lims,
+    flow_cell_id: str,
+    flow_cell_sample_type: Type[FlowCellSample],
+) -> Iterable[FlowCellSample]:
+    LOG.info(f"Fetching samples from lims for flowcell {flow_cell_id}")
```

Comment:
```suggestion
    LOG.info(f"Return samples from LIMS for flow cell {flow_cell_id}")
```

Reply:
I am thinking if it is better to leave this as a verb in present participle as it describes an ongoing process